client-cli/README.md

```diff
@@ -16,10 +16,14 @@ For example, to list dataspace participants:
 
 ```
 java -jar client-cli/build/libs/registration-service-cli.jar \
-  -s=http://localhost:8182/authority \
+  -dd=did:web:dataspaceauthoritydid \
```

Comment:
I would discourage using two-letter option names. In Unix practice they are either one letter or long names

Reply:
so what would be better, eg.
-c for client did
-d for dataspace did

or 

-client-did
-dataspace-did

-clientdid
-dataspacedid

Reply:
you can actually provide both like here: https://github.com/agera-edc/IdentityHub/blob/e456e4bfad2dc15f97c22a16eb4485cb43ff179f/client-cli/src/main/java/org/eclipse/dataspaceconnector/identityhub/cli/AddVerifiableCredentialCommand.java#L35

Reply:
ok, nice, so I'll go then with 
-c for client did
-d for dataspace did
and 
--client-did
--dataspace-did

as alternatives thanks!